src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/21c93fc1d0.js" crossorigin="anonymous"></script>
+    <title>To-do-list</title>
+</head>
+<body>
+    <div class="container d-flex justify-content-center align-items-center ">
+        <div class="row justify-content-center w-100 ">
+
+            <div class="card col-lg-6 col-md-8 col-12 ">
+                <div class="card-body">
+                    <h5 class="card-title d-flex justify-content-between align-items-center ">Today's To Do<span>
+                        <i class="fas fa-sync" id="refresh"></i>
+                    </span></h5>
+                    <input type="text" class="form-control my-3"  aria-label="Default" id="task"
+                        aria-describedby="inputGroup-sizing-default" placeholder="Add to your lists">
+                    <ul id="items" class="p-1">
+                    <!-- placeholder for to-do-items -->
+                    </ul>
+                    <div class="text-center">
+                        <a href="#" class="text-secondary clear" id="clearCompleted">Clear all Completed</a>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+
+    </div>
```

Comment:
As a best practice, it is recommended to remove any empty lines in the HTML code to improve readability and minimize the file size. This will not only result in more efficient rendering of the web page but also make it easier for other developers to collaborate and maintain the codebase in the future.